### PR TITLE
New types

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin @sdroege 

It's required to be able to have the `MountOperation` working normally in gtk.

### CODE REVIEW

**Comment by EPashkin** [comment]
This conflicting with #69

**Comment by GuillaumeGomez** [comment]
Indeed. If #69 is far from being merged, do you mind removing these new types from it please so I can move forward on gtk?

File: src/auto/mount_operation.rs [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+    fn connect_aborted<F: Fn(&Self) + 'static>(&self, f: F) -> SignalHandlerId {
+        unsafe {
+            let f: Box_<Box_<Fn(&Self) + 'static>> = Box_::new(Box_::new(f));
+            connect(self.to_glib_none().0, "aborted",
+                transmute(aborted_trampoline::<Self> as usize), Box_::into_raw(f) as *mut _)
+        }
+    }
+
+    //fn connect_ask_password<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId {
+    //    Ignored flags: Gio.AskPasswordFlags
```
Comment (sdroege) [inline src/auto/mount_operation.rs]: Just add them :)

File: src/auto/mount_operation.rs [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+    //}
+
+    //fn connect_ask_question<Unsupported or ignored types>(&self, f: F) -> SignalHandlerId {
+    //    Empty ctype choices: *.CArray TypeId { ns_id: 0, id: 28 }
```
Comment (sdroege) [inline src/auto/mount_operation.rs]: That's just a GStrv, we support that?!

File: src/auto/file.rs [hunk 3]
```diff
@@ -1027,11 +1027,11 @@ impl<O: IsA<File>> FileExt for O {
...
+    //fn start_mountable<'a, 'b, 'c, P: Into<Option<&'a MountOperation>>, Q: Into<Option<&'b Cancellable>>, R: Into<Option<&'c /*Unimplemented*/AsyncReadyCallback>>, S: Into<Option</*Unimplemented*/Fundamental: Pointer>>>(&self, flags: /*Ignored*/DriveStartFlags, start_operation: P, cancellable: Q, callback: R, user_data: S) {
     //    unsafe { TODO: call ffi::g_file_start_mountable() }
     //}
 
-    //fn stop_mountable<'a, 'b, 'c, P: Into<Option<&'a /*Ignored*/MountOperation>>, Q: Into<Option<&'b Cancellable>>, R: Into<Option<&'c /*Unimplemented*/AsyncReadyCallback>>, S: Into<Option</*Unimplemented*/Fundamental: Pointer>>>(&self, flags: /*Ignored*/MountUnmountFlags, mount_operation: P, cancellable: Q, callback: R, user_data: S) {
+    //fn stop_mountable<'a, 'b, 'c, P: Into<Option<&'a MountOperation>>, Q: Into<Option<&'b Cancellable>>, R: Into<Option<&'c /*Unimplemented*/AsyncReadyCallback>>, S: Into<Option</*Unimplemented*/Fundamental: Pointer>>>(&self, flags: /*Ignored*/MountUnmountFlags, mount_operation: P, cancellable: Q, callback: R, user_data: S) {
```
Comment (sdroege) [inline src/auto/file.rs]: @antoyo More async functions that are not called _async

**Comment by GuillaumeGomez** [comment]
Ok, I think this PR is ready. Just waiting to know if @EPashkin's PR will take long before getting merged or not.

**Comment by EPashkin** [comment]
It seems don't conflict with my PR, so I rebase it after this merged

**Comment by GuillaumeGomez** [comment]
Thanks! I'm just waiting for the CIs and then I merge.

**Comment by EPashkin** [comment]
@GuillaumeGomez 
CI passed.
Also I reminder you about docs.

**Comment by GuillaumeGomez** [comment]
What about docs?

**Comment by EPashkin** [comment]
Please, regen docs for gtk-rs.

**Comment by GuillaumeGomez** [comment]
Sure, I just finish my gtk PR first and here we go.

**Pull request merged**